pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -138,18 +139,21 @@ func EnforceHostNetwork() bool {
 }
 
 func SetRevisionHistoryLimit(data map[string]string) {
-	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultValue)
-	if strval != "" {
-		numval, err := strconv.ParseInt(strval, 10, 32)
-		if err != nil {
-			logger.Warningf("failed to parse value %q for %q. assuming default value.", strval, revisionHistoryLimitSettingName)
-			revisionHistoryLimit = nil
-			return
-
-		}
-		limit := int32(numval)
-		revisionHistoryLimit = &limit
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionHistoryLimitDefaultStr)
+	var limit int32
+	if strval == "" {
```

Comment:
There is inconsistency between how defaults are coded. The function uses `revisionHistoryLimitDefaultStr`, but immediately after, there is `if strval == ""` which is checking for the default string. Since the default is empty, IMO, it's easier to read the code if we get rid of `revisionHistoryLimitDefaultStr` and just use `""`.

Reply:
good point! fixed.